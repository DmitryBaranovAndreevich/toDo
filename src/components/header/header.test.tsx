import renderer from 'react-test-renderer';
import Header from './index';

it('Header component is rendered without errors', () => {
  const header = renderer.create(<Header />).toJSON();
  expect(header).toMatchSnapshot();
});
