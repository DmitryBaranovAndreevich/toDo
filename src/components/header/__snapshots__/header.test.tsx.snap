// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component is rendered without errors 1`] = `
<div
  className="container"
>
  <img
    alt="Eisenhower matrix"
    className="image"
    src="eisenhower-matrix.jpg"
  />
  <a
    className="info"
    href="https://goal-life.com/page/method/matrix-eisenhower"
  >
    Want more details?
  </a>
</div>
`;
